docker/server.xml

```
@@ -108,7 +108,7 @@
     -->
 
     <!-- Define an AJP 1.3 Connector on port 8009 -->
-    <Connector port="8009" protocol="AJP/1.3" redirectPort="8443" />
+    <Connector port="8009" protocol="AJP/1.3" redirectPort="8443" secret="xyz123" />
```

Hm, a "magical" :imp: `secret`... I doubt anyone uses the AJP Connector with Tomcat in a Container, and for better security than shipping with a default password (bad best practice) I suggest that we just remove (or at least comment out) this `Connector` entirely here, what do you think?

---

Thanks - that makes sense. I only added it to suppress an error message from Tomcat 9. But you're right that a better approach would be to disable the connector altogether. Will do that next. 

And the magical secret was reuse of the keystore password in the same config file :-)